### PR TITLE
Adding calcAvg to the problems page 

### PR DESCRIPTION
**Pull request by lalaAdasia00**
Adding problems to increase visibility of pagination

### CODE REVIEW

**Review by ifeomi — APPROVED** [review]

File: problems/calcAvg.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  - assert(calcAvg([5, 123, 12, 24]) === 41);
+  - assert(calcAvg([-12, 23, 45, 67, -89]) === 6.8);
+  - assert(calcAvg([0, 789, 456, 123, -159]) === 241.8);
+solution: |
+  function calcAvg(arr){
+    var sum = 0;
+    for( var i = 0; i < arr.length; i++ ){
+      sum += parseInt( arr[i], 10 ); 
+    }
+    var avg = sum/arr.length;
```
Comment (chronologos) [inline problems/calcAvg.yaml]: how about division by zero?

File: problems/calcAvg.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  function calcAvg(arr) {
+    return num;
+  }
+
+  assert(calcAvg([5, 123, 12, 24]) === 41);
+  assert(calcAvg([-12, 23, 45, 67, -89]) === 6.8);
+  assert(calcAvg([0, 789, 456, 123, -159]) === 241.8);
+  alert('Passed all test cases!');
+tests:
+  - assert(calcAvg([5, 123, 12, 24]) === 41);
```
Comment (chronologos) [inline problems/calcAvg.yaml:28]: include edge test cases
e.g. 0 and 1 element.

**Review by chronologos — CHANGES_REQUESTED** [review]

File: problems/calcAvg.yaml (hunk 2 above)
Comment (lalaAdasia00) [reply problems/calcAvg.yaml:23]: I will ask the user to return null

File: problems/calcAvg.yaml (hunk 1 above)
Comment (lalaAdasia00) [reply problems/calcAvg.yaml]: I will ask the user to return null

File: problems/calcAvg.yaml (hunk 2 above)
Comment (lalaAdasia00) [reply problems/calcAvg.yaml:28]: Will do :)

**Review by chronologos — APPROVED** [review]

**Pull request merged**